### ISSUE TITLE
http and websocket on same route

### ISSUE THREAD

**Issue by illusionTBA** [elysiajs/elysia#263]
Hello, recently ive needed to implement an http route and a websocket on the same route but i cant figure out how to do it or if its even possible with elysia when ive done it with bun's built in `serve`

the following code is an example of what im trying to achieve:
```ts
 const app = new Elysia({
        name: 'bare',
        seed: {
            prefix,
        }
    })
  
  app.all("/v3/", (ctx) => {
      return new Response("Hello from http")
  })

  app.ws("/v3/",  {
      message(ws, message) {
          ws.send("pong")
      }
  })

```
^ this is all within a plugin
Also it seems the websocket takes priority over the HTTP route as when i go to `http://localhost:3000/v3/` i get `Expected a websocket connection`
any help on this will be much appreciated

**Comment by SaltyAom** [comment]
Should be fixed on [fe95cb8](https://github.com/elysiajs/elysia/commit/fe95cb8eb30438779d47e45fcf6445aa9780e08e)

I have also added a test case for this.

**Comment by illusionTBA** [comment]
tysm, works like a charm now!
